Foundation/NSURLSession/NSURLSessionConfiguration.swift

```diff
@@ -47,6 +47,7 @@ open class URLSessionConfiguration : NSObject, NSCopying {
         self.urlCredentialStorage = nil
         self.urlCache = nil
         self.shouldUseExtendedBackgroundIdleMode = false
+	self.protocolClasses = [_HTTPURLProtocol.self]
```

Comment:
@ianpartridge  Not sure why this is not aligned here in this view. But I see that in the original file it is aligned with previous statements.